OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitMobileNotSupportedFragment.java

```diff
@@ -9,40 +9,20 @@
 import org.edx.mobile.R;
 import org.edx.mobile.model.course.BlockType;
 import org.edx.mobile.model.course.CourseComponent;
-import org.edx.mobile.services.ViewPagerDownloadManager;
 import org.edx.mobile.util.BrowserUtil;
 
-/**
- *
- */
 public class CourseUnitMobileNotSupportedFragment extends CourseUnitFragment {
-
     /**
      * Create a new instance of fragment
      */
     public static CourseUnitMobileNotSupportedFragment newInstance(CourseComponent unit) {
         CourseUnitMobileNotSupportedFragment f = new CourseUnitMobileNotSupportedFragment();
```

Comment:
Need to refactor variable `f` to meaning full name, and should be declare as `final`